fixtures/js-stubs/projectAlertRule.ts

```diff
@@ -23,6 +23,7 @@ export function ProjectAlertRule(params: Partial<IssueAlertRule> = {}): IssueAle
     projects: [],
     snooze: false,
     frequency: 1,
+    status: 0,
```

Comment:
Why is this only 0? Shouldn't it be 0 or 1?

Reply:
this is the stub we use in tests, setting it to enabled

Reply:
oooh ok